cmake/linux/installer

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+set -e
+[ "$1" ]
+
+VERSION="$1"
+DIR="$HOME"/.lmms-$VERSION
```

Comment:
I like the .run idea too, as the Qt standalone installer works: if you run it as a "normal" user it installs in your home directory and if you run it as root it installs in /opt. The best of the two worlds...